.pylintrc

```diff
@@ -60,89 +60,97 @@ confidence=
 # --enable=similarities". If you want to run only the classes checker, but have
 # no Warning level messages displayed, use "--disable=all --enable=classes
 # --disable=W".
-disable=print-statement,
-        parameter-unpacking,
-        unpacking-in-except,
-        old-raise-syntax,
-        backtick,
-        long-suffix,
-        old-ne-operator,
-        old-octal-literal,
-        import-star-module-level,
-        non-ascii-bytes-literal,
-        raw-checker-failed,
+#
+# Commented-out options were removed in PR4942
+# disable=print-statement,
+#         parameter-unpacking,
+#         unpacking-in-except,
+#         old-raise-syntax,
+#         backtick,
+#         long-suffix,
+#         old-ne-operator,
+#         old-octal-literal,
+#         import-star-module-level,
+#         non-ascii-bytes-literal,
+#         apply-builtin,
+#         basestring-builtin,
+#         buffer-builtin,
+#         cmp-builtin,
+#         coerce-builtin,
+#         execfile-builtin,
+#         file-builtin,
+#         long-builtin,
+#         raw_input-builtin,
+#         reduce-builtin,
+#         standarderror-builtin,
+#         unicode-builtin,
+#         xrange-builtin,
+#         coerce-method,
+#         delslice-method,
+#         getslice-method,
+#         setslice-method,
+#         no-absolute-import,
+#         old-division,
+#         dict-iter-method,
+#         dict-view-method,
+#         next-method-called,
+#         metaclass-assignment,
+#         indexing-exception,
+#         raising-string,
+#         reload-builtin,
+#         oct-method,
+#         hex-method,
+#         nonzero-method,
+#         cmp-method,
+#         input-builtin,
+#         round-builtin,
+#         intern-builtin,
+#         unichr-builtin,
+#         map-builtin-not-iterating,
+#         zip-builtin-not-iterating,
+#         range-builtin-not-iterating,
+#         filter-builtin-not-iterating,
+#         using-cmp-argument,
+#         eq-without-hash,
+#         div-method,
+#         idiv-method,
+#         rdiv-method,
+#         exception-message-attribute,
+#         invalid-str-codec,
+#         sys-max-int,
+#         bad-python3-import,
+#         deprecated-string-function,
+#         deprecated-str-translate-call,
+#         deprecated-itertools-function,
+#         deprecated-types-field,
+#         next-method-defined,
+#         dict-items-not-iterating,
+#         dict-keys-not-iterating,
+#         dict-values-not-iterating,
+#         deprecated-operator-function,
+#         deprecated-urllib-function,
+#         xreadlines-attribute,
+#         deprecated-sys-function,
+#         exception-escape,
+#         comprehension-escape,
+#
+# This option was removed in PR 3571
+# See also: https://pylint.pycqa.org/en/latest/whatsnew/2/2.6/summary.html#summary-release-highlights
+#         bad-continuation,
```

Comment:
Do we need to save this as a comment-out?

Reply:
No, we can remove this. I just wanted you to see why I deleted those, hence the comments.